main.go

```diff
@@ -50,7 +65,8 @@ func main() {
 	viper.AddConfigPath(home)
 	viper.AddConfigPath(".")
 	viper.SetConfigName(".zoom-slack-status")
-	viper.SetDefault("interval", 60*time.Second)
+
+	viper.SetDefault("Interval", defaultInterval)
```

Comment:
Looks like `interval` is lowercased on L35, will this break?

```suggestion
	viper.SetDefault("interval", defaultInterval)
```

Reply:
This will not break, because [Viper configuration keys are case-insensitive](https://pkg.go.dev/github.com/spf13/viper#section-readme:~:text=Important%3A%20Viper%20configuration%20keys%20are%20case%20insensitive.). However, it would be best not to rely on this so I will update this as per your suggestion.